Update “Contribute to the community” on TensorFlow.org (#2034)
tensorflow/docs: site/en/community/contribute/community.md

The TensorFlow Forum is organized by categories, subcategories and tags. We encourage you to create an account and follow categories and tags of interest. When you create a new post, select the most appropriate [category or subcategory](https://discuss.tensorflow.org/categories) and [tags](https://discuss.tensorflow.org/tags) to help other users find your topic.

For more information on Discourse features, read the [Discourse New User Guide](https://meta.discourse.org/t/discourse-new-user-guide/96331).

### Become a Forum expert

Discourse uses [trust levels](https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/) to reward increasing levels of participation in the forum. The Forum facilitates learning by doing, letting you to collect [badges](https://discuss.tensorflow.org/badges) that are displayed on your profile. This is a great way to be recognized for helping fellow community members. The more you invest in helping community members, the more badges and forum tools you will unlock.

> Certain groups, such as TensorFlow Team members and Machine Learning GDEs display a special icon for easier identification.

Reviewer: Nit (add a comma)

```suggestion
Certain groups, such as TensorFlow Team members and Machine Learning GDEs, display a special icon for easier identification.
```

The primary communication about work on TensorFlow happens in the [TensorFlow repositories on GitHub](https://github.com/tensorflow). This is the place to discuss bugs, new features, and in-progress work.

### Mailing lists

> Most communication happens on the TensorFlow Forum, however the following mailing lists are still used for announcements and contributor conversation. They are not intended to provide technical support.

Reviewer: ```suggestion
Most communication happens on the TensorFlow Forum. The following mailing lists are still used for announcements and contributor conversations (note that they are not intended to provide technical support).
```

LMKWYT

@theadactyl @kristenrq

Reviewer 2: Nit: I like breaking the first sentence up, but I would leave the last sentence as its own sentence ("Note that...", as I think the point is more than parenthetical.

Reviewer: Done @wolffg

*   [announce@tensorflow.org](mailto:announce@tensorflow.org) — All major releases and important announcements are sent to this mailing group. We recommend that you join this list if you depend on TensorFlow in any way.
*   [developers@tensorflow.org](mailto:developers@tensorflow.org) — Discussion for developers who are contributing to TensorFlow.

### Blog and social media

> The [TensorFlow Blog](http://blog.tensorflow.org/) is full of great content both from our team at Google and the broader community. We'd love to hear what you have to say, so if you would like to submit an article for review, please contact the TensorFlow Developer Relations team.

Reviewer: ```suggestion
The [TensorFlow Blog](http://blog.tensorflow.org/) is full of great content both from our team at Google and the broader community. We'd love to hear what you have to say, so if you would like to submit an article for review, please contact us at tensorflow-blog@google.com. Note that we receive many great submissions, and setting expectations, we can only publish a few.
```

cc @theadactyl @random-forests @wolffg  as discussed with the team

Follow the [#TFCommunitySpotlight](https://twitter.com/hashtag/TFCommunitySpotlight?src=hashtag_click) hashtag and find out more about past winners [here](https://blog.tensorflow.org/2020/11/tensorflow-community-spotlight-program-update.html).

## User groups

[TensorFlow User Groups](https://www.tensorflow.org/community/groups) (or TFUGs, for short) are local communities of developers and researchers around the world. If you don’t have a TFUG in your country or city, we encourage you to start one by reaching out to [tfug-help@tensorflow.org](mailto:tfug-help@tensorflow.org).

## Events

The TensorFlow team hosts and supports events all around the world! If your TFUG is planning an upcoming event or meetup, please let our Community know by posting about it on the TensorFlow Forum under the [Events category](https://discuss.tensorflow.org/c/events/27).

If you’ve already held your event, please share feedback with us [here](https://docs.google.com/forms/d/e/1FAIpQLSdvb8c2ZFXxS05aX6dpUVZlfYA0WsFFq-sUAzjiohVKAQ1RLw/viewform)! Feel free to share recaps and recordings on the Forum as well.

## TensorFlow Certificate program

> Demonstrate your proficiency in using TensorFlow to solve deep learning and ML problems. Advance your career and connect with developers who also have passed the exam on the [Certificate Network](https://www.tensorflow.org/certificate-network).

Reviewer: ```suggestion
You can demonstrate your proficiency in using TensorFlow to solve deep learning and ML problems with the [TensorFlow Certificate](https://www.tensorflow.org/certificate) program. And, with the [Certificate Network](https://www.tensorflow.org/certificate-network), you can advance your career and find developers who also have passed the exam.
```

cc @theadactyl, as discussed with the team

Reviewer: cc @random-forests @wolffg @JosBecker

> Stipends are available. See [more information](https://www.tensorflow.org/certificate) on the TensorFlow Certificate program.

Reviewer: ```suggestion
Stipends are available. [Learn more](https://www.tensorflow.org/certificate) about the TensorFlow Certificate program.
```

Suggestion: use more accessibility/a11y-friendly language (e.g. "see" -> "learn more") cc @heyawhite
